php/class-wp-cli.php

```diff
@@ -233,7 +233,7 @@ public function assign_user_to_coauthor( $args, $assoc_args ) {
 		}
 
 		$post_types = implode( "','", $coauthors_plus->supported_post_types );
-		$posts      = $wpdb->get_col( $wpdb->prepare( "SELECT ID FROM $wpdb->posts WHERE post_author=%d AND post_type IN (%s)", $user->ID, $post_types ) );
+		$posts      = $wpdb->get_col( $wpdb->prepare( "SELECT ID FROM $wpdb->posts WHERE post_author=%d AND post_type IN ('" . $post_types . "')", $user->ID ) );
```

Comment:
Do you mind throwing a `// phpcs:ignore` comment for the ruleset this does not need? Also, minor, but let's use interpolation over concatenation for better readability.

Reply:
@rebeccahum changes ready for review